testsuite/domain/src/test/java/org/jboss/as/test/integration/domain/HostExcludesTestCase.java

```diff
@@ -210,7 +210,8 @@ private enum ExtensionConf {
                 "org.wildfly.extension.microprofile.lra-participant",
                 "org.wildfly.extension.microprofile.telemetry"
         ), true),
-        CURRENT(MAJOR, WILDFLY_28_0, getCurrentAddedExtensions(), getCurrentRemovedExtensions(), true);
+        WILDFLY_29_0("WildFly29.0", WILDFLY_28_0, true),
+        CURRENT(MAJOR, WILDFLY_29_0, getCurrentAddedExtensions(), getCurrentRemovedExtensions(), true);
```

Comment:
Just for the record, it is good to have it adjusted now (even if the product version is still 29) but it's not strictly necessary to do it on each new release if we have not added/removed extensions on the current release.

The test won't fail if we have moved the product version to the next one (e.g. 30) but we have not added/removed extensions on the current release (e.g. 29).

Reply:
Without this change, it seems that all full-integration tests on wildfly-core are failing, e.g. https://ci.wildfly.org/viewLog.html?buildId=381306&buildTypeId=WildFlyCore_PullRequest_WildFlyCoreFullIntegrationLinuxJdk11
Was I mistaken?

Reply:
> The test won't fail if we have moved the product version to the next one (e.g. 30) but we have not added/removed extensions on the current release (e.g. 29).

Ah - I see.  So, were the product version changed (I assume, via an update to org.wildfly.core:wildfly-version + a wildfly-core upgrade?), this PR would not have been strictly necessary.  Correct?

Reply:
Ups, sorry, I missed this comment and my review was incorrect since there were extensions removed in the current release (WF29).